src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/CalendarDatePickerDateChangedEventArgs.cs

```diff
@@ -16,8 +16,8 @@ public  partial class CalendarDatePickerDateChangedEventArgs
 				throw new global::System.NotImplementedException("The member DateTimeOffset? CalendarDatePickerDateChangedEventArgs.NewDate is not implemented in Uno.");
 			}
 		}
-		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#endif
+#if false || false || false || false || false
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826528)

Reply:
This is style related... implemented methods can be:
a) removed from Generated completely, or
b) made #if false, or
c) made #if with false for every platform.

In many Generated files, "#if false || false || false || false || false" can be seen. I thought it was correct style. If it should be corrected, please state correct version (a, b or c).

Reply:
Don't worry, this is fine... we're testing an automated code-checking tool, but it's tending to generate false positives like this.